src/etc/telegraf/custom_scripts/vyos_services_input_filter.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import time
+from vyos.config import Config
+from psutil import process_iter
+# Availible services
+services = {
+    "service pppoe"  : "pppoe.conf",
+    "protocols bgp"  : "/usr/lib/frr/bgpd",
+    "protocols ospf" : "/usr/lib/frr/ospfd",
+    "protocols rip"  : "/usr/lib/frr/ripd",
+}
+# Configured services
+conf_services = {
+    '/usr/lib/frr/zebra'   : 0,
+    '/usr/lib/frr/staticd' : 0,
+}
+# Get configured service and create list to check if proucess running
+for service in services:
+    if Config().exists_effective(service):
+        conf_services[services[service]] = 0
+active_pocess = []
+# Get running process object
+active_pocess_obj = process_iter()
+for p in active_pocess_obj:
+    for conf_service in conf_services:
+        if conf_service in p.cmdline():
+            # print(conf_service)
+            conf_services[conf_service] = 1
+for conf_service in conf_services:
+    # Printing data in Influx Line Protocol format
+    print("vyos_services,service=\""+conf_service+"\" status="+str(conf_services[conf_service])+"i "+str(int(time.time()))+"000000000")
```

Comment:
This should also use a format string.

Also, what is `str(int(time.time()))+"000000000")` supposed to do? If it's to signify specific precision, this can also be done with integer formatting in format strings.

Reply:
Replaced to `fstring` format